### PR TITLE
Meta-class: The N+S redesign.

### PR DESCRIPTION
**Pull request by Zevotech**
## About The Pull Request
Remaps the Meta-class Freighter (for the second time) to have an improved layout and N+S affiliation. Includes a beautiful crew area and quartermaster's office, a spacious cargo bay, substandard turrets and the dust and junk you know and love from the original Meta-class.

metaingame

StrongDMM-2024-07-13 00 19 28
StrongDMM-2024-07-13 00 19 32

- [ ] HEY I NEED N+S LOGISTICS PAPERWORK/PAMPHLETS FOR THIS. PLEASE DM ME SOME OR ILL HAVE TO MAKE THEM MYSELF AND THEY'LL PROBABLY SUCK!

## Why It's Good For The Game
Better NT subfaction representation + I got told to do my job and I did it.
The previous iteration of the meta definitely was not my best work and was mostly done to bring it up to standard rather than properly make it an NT ship, which led to it being pretty unpopular despite my wishes. This version should be a lot more fun to play. Hopefully.

## Changelog

:cl:
add: Remapped the Meta class to be under N+S affiliation.
/:cl:

### CODE REVIEW

File: _maps/configs/nanotrasen_meta.json [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-	"prefix": "NTSV",
+	"prefix": "NSSV",
 	"map_short_name": "Meta-class",
-	"description": "The Meta-class is a medium freight vessel designed for comfort and sustainability over long journeys of delivering cargo. Most, if not all of these ships are no longer under the direct management of the Company, leaving regulation and professionalism at the behest of individual captains. They can be found running independent contracts, delivering cargo, smuggling illicit goods, and generally trying to stay afloat ever since the collapse of Nanotrasen's operations in the frontier.",
+	"description": "The Meta-class is the mainline freighter of N+S Logistics, boasting incredible cargo capacity, spacious crew areas, and law-abiding levels of speed. It has been through a substantial redesign compared to it's pre-war namesake, aiming to reduce accidents and increase supply throughput to make up for post-war defecits. The Meta-class frigate can most often be found in the frontier transporting cargo between outposts, delivering mail, and selling N+S goods and merchandise. N+S is not liable for any health issues or addictions that may occur while working aboard this vessel.",
```
Comment (SomeguyManperson) [inline _maps/configs/nanotrasen_meta.json]:
```suggestion
	"description": "The Meta-class is the mainline freighter of N+S Logistics, boasting incredible cargo capacity, spacious crew areas, and law-abiding levels of speed. It has been through a substantial redesign compared to its pre-war namesake, aiming to reduce accidents and increase supply throughput to make up for post-war defecits. The Meta-class frigate can most often be found in the frontier transporting cargo between outposts, delivering mail, and selling N+S goods and merchandise. N+S is not liable for any health issues or addictions that may occur while working aboard this vessel.",
```

**Comment by Leon9621** [comment]
Heya, just played a round on this! Pretty serious issue, it seems like the engines aren't wired properly at the start, they won't connect and the ship can't fly. Other than that though, I couldn't notice any other issues. Good work!

I don't know exactly how they were fixed, an admin came in to do the repair work. Terribly unhelpful, I know

**Comment by Zevotech** [comment]
yea i was notified during the round but wasnt home to fix it. should be fixed in a moment once i push the changes and tell someone to update tms

**Comment by Zevotech** [comment]
Closing this until our cargo mechanics get some work, and because I don't want it merged until I put some more things on it. Will reopen at a later date, probably.

**Pull request closed without merging**